lib/attributor/types/csv.rb

```diff
@@ -29,5 +29,9 @@ def self.example(context=nil, options: {})
       return collection.join(',')
     end
 
+    def self.family
```

Comment:
actually, I think this should be array....since this is not about the acceptable encoding, but more about the native storage/type... no? CSV is really an array at heart, which can be loaded from a string...which is very similar to say, a Hash/Model which are structs with properties, albeit they can be loaded from a string too (i.e., JSON)